tests/php/modules/shortcodes/test-class.inline-pdf.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Unit test for Inline PDF embeds.
+ *
+ * @package Jetpack
+ * @since   8.4
+ */
+class WP_Test_Jetpack_Shortcodes_Inline_Pdfs extends WP_UnitTestCase {
+
+	/**
+	 * Unit test for Inline PDF embeds.
+	 *
+	 * @author lancewillett
+	 * @covers ::jetpack_inline_pdf_embed_handler
+	 * @since  8.4
+	 */
+	public function test_shortcodes_inline_pdf() {
+		global $post;
+
+		$url  = 'https://jetpackme.files.wordpress.com/2017/08/jetpack-tips-for-hosts.pdf';
+		$post = $this->factory()->post->create_and_get( array( 'post_content' => $url ) );
+
+		setup_postdata( $post );
+
+		// Test HTML version.
+		ob_start();
+		the_content();
+		$actual = ob_get_clean();
+		$this->assertContains(
+			sprintf(
+				'<p><object data="%1$s" type="application/pdf" width="100 height="800"><p><a href="%1$s">%1$s</a></p></object></p>' . "\n",
```

Comment:
The test is failing right now:

```
1) WP_Test_Jetpack_Shortcodes_Inline_Pdfs::test_shortcodes_inline_pdf
Failed asserting that '<p><object data="https://jetpackme.files.wordpress.com/2017/08/jetpack-tips-for-hosts.pdf" type="application/pdf" width="100%" height="800"><p><a href="https://jetpackme.files.wordpress.com/2017/08/jetpack-tips-for-hosts.pdf">https://jetpackme.files.wordpress.com/2017/08/jetpack-tips-for-hosts.pdf</a></p></object></p>
' contains "<p><object data="https://jetpackme.files.wordpress.com/2017/08/jetpack-tips-for-hosts.pdf" type="application/pdf" width="100 height="800"><p><a href="https://jetpackme.files.wordpress.com/2017/08/jetpack-tips-for-hosts.pdf">https://jetpackme.files.wordpress.com/2017/08/jetpack-tips-for-hosts.pdf</a></p></object></p>
".
```

I pushed fc617a919939a6668fe5f079c5f1de983bdc57aa to try to fix this.